### PR TITLE
BRITECHSRM-19 - Connect F# to MySQL DB

### PR DESCRIPTION
**Pull request by lclarksl**
Note that the connection string is now committed without a password. This is because security is being handled by AWS security groups so requests are only accepted from Scott Logic IPs (for development) and the EC2 instance this will run on.

### CODE REVIEW

File: Speakers/Controllers.fs [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+
+type TalkOutlinesController() =
+    inherit ApiController()
+
+    member x.Get() =
+        printfn "Received GET request for talk outlines"
+        let talkOutlines = getAllTalkOutlines
+        x.Request.CreateResponse(talkOutlines)
+
+    member x.Get(id:int) =
```
Comment (csscottlogic) [inline Speakers/Controllers.fs]: Do we need this?

**Comment by csscottlogic** [comment]
Apart from maybe not needing to get a particular talk outline (yet), :shipit:

**Comment by csscottlogic** [comment]
We now need to get the data from Trello into the DB :-)

**Comment by jebbinSCL** [comment]
:shipit:

**Pull request merged**